pkg/hsm/awscloudhsm/cloudhsm.go

```diff
@@ -32,20 +31,78 @@ func (c *Client) CreateRSAKeyPair(label string, hsmCreds hsm.Credentials) (publi
 		fmt.Sprintf("HSM_PASSWORD=%s", hsmCreds.Password),
 		fmt.Sprintf("HSM_IP=%s", hsmCreds.IP),
 	)
-	cert, err := cmd.Output()
+	out, err := cmd.Output()
 	if err != nil {
 		return nil, fmt.Errorf("failed to generate rsa key for %s: %s", label, err)
 	}
-	if !strings.Contains(string(cert), "--BEGIN CERTIFICATE--") {
```

Comment:
Is this no longer an issue?

Reply:
Yeah this is ok now ... the cloudhsmtool correctly returns exit 1 on error (I think `mdgen` still happily exits 0 on error tho)